cfme/services/myservice/ui.py

```diff
@@ -55,9 +55,11 @@ class MyServicesView(BaseLoggedInPage):
 
     @property
     def in_myservices(self):
+        # Slicing in currently_selected is workaround for BZ-1733489
```

Comment:
I think i may be good to do BZ(1733489).blocks here? Thoughts?

Reply:
why we need blocks?
we are just limiting to index 1. If BZ fix anyway it will work fine.